data/scenarios/Tutorials/bind2.yaml

```diff
@@ -77,10 +77,12 @@ robots:
       attr: robot
     devices:
       - 3D printer
+      - logger
```

Comment:
We should test that all tutorial scenarios have a logger otherwise players will miss out on key help features. :scream: